docs/howto/installing-local-extensions.md

```diff
@@ -80,21 +125,70 @@ paths:
     themebase: public/theme
     files: public/files
     view: public/bolt-public/view
+
 extensions:
-    - MyextOne\LocalExtensionOne
-    - MyextTwo\LocalExtensionTwo
-    - MyextThree\LocalExtensionThree
+    - Myextension\LocalExtensionOne
+    - Listing\ListingExtension
 ```
 
-Note that each class defined as above eg: `MyextOne\LocalExtensionOne` must have been
-setup using one of the methods defined above.
+Note that each class defined as above eg: `MyextOne\LocalExtensionOne` must
+have been setup using one of the methods defined above.
+
+If you use a PHP autoloader instead of the yaml style one above then you can
+add your extensions in PHP code in either `.bolt.php` or in your own bootstrap.
```

Comment:
> PHP autoloader

Everyone uses a PHP autoloader 😂 . It's basically just saying you can add extensions in `.bolt.php` if you use that instead of `.bolt.yml`. Which makes it redundant with the examples below.